tensorhive/config.py

```diff
@@ -178,7 +197,39 @@ class PROTECTION_SERVICE:
     ENABLED = config.getboolean(section, 'enabled', fallback=True)
     UPDATE_INTERVAL = config.getfloat(section, 'update_interval', fallback=2.0)
     NOTIFY_ON_PTY = config.getboolean(section, 'notify_on_pty', fallback=True)
-
+    NOTIFY_VIA_EMAIL = config.getboolean(section, 'notify_on_pty', fallback=True)
+
+class MAILBOT:
+    mailbot_config = ConfigLoader.load(CONFIG_FILES.MAILBOT_CONFIG_PATH, displayed_title='mailbot')
+    section = 'general'
+    INTERVAL = mailbot_config.getfloat(section, 'interval', fallback=10.0)
+    NOTIFY_INTRUDER = mailbot_config.getboolean(section, 'notify_intruder', fallback=True)
+    NOTIFY_ADMIN = mailbot_config.getboolean(section, 'notify_admin', fallback=True)
+    ADMIN_EMAIL = mailbot_config.get(section, 'admin_email', fallback='')
+
+    # FIXME Not sure if this should be required
+    section = 'smtp'
+    SMTP_LOGIN_ENV = mailbot_config.get(section, 'email_env_var')
```

Comment:
Why do we specify email_env_var in config instead of simply specifying email and password in config?

Reply:
Email address: maybe yes..., but I have mixed feelings about the password...
I thought that in order to be concise these should be treated equally.

My first experience with such dilemma was about 2 years ago with Docker and the answer was:
![image](https://user-images.githubusercontent.com/12485656/51943123-03520500-2419-11e9-9c62-a7264133c645.png)

All in all, we can keep the password in `.ini` files as long as we're sure that noone, besides admin, can open this file (admin should take care of it by himself).

Reply:
Maybe we could keep the password in the mailbot config, but make sure to protect the file while creating it:
chmod go-r ~/.config/TensorHive/mailbot_config.ini